requirements/test_requirements.txt

```diff
@@ -8,3 +8,4 @@ GPy>=1.9.6
 matplotlib
 mxnet>=1.3
 mxboard>=0.1.0
+mock
```

Comment:
Please specify a version number. Just use the version that you have tested working.

Reply:
mock>=3.0.5  or mock==3.0.5?

Reply:
Please use ```mock>=3.0.5``` as we will not actively maintain the dependency consistence. Setting exact version numbers can easily result in complicated installation failure.